File: Lua/Daily001_easy/UserInput.lua
-- USER INPUT 
--
-- challenge #1 (easy)
-- https://www.reddit.com/r/dailyprogrammer/comments/pih8x/easy_challenge_1/ 
--
--
-- sarthak7u@gmail.com
--

print("What is your Name?")
local name = io.read()

print("What is your age?")
local age = io.read()

print("What is your username?")
local uname = io.read()

local output = "your name is " .. name .. ", you are " .. age .. " years old, and your username is " .. uname

print(output)

local f = io.open('output.txt', 'w')
if f then
  io.output(f)
  io.write(output)
  io.close(f)
end

File: Dart/Daily001_easy/UserInput.dart
/** 
 * USER INPUT 
 *
 * challenge #1 (easy)
 * https://www.reddit.com/r/dailyprogrammer/comments/pih8x/easy_challenge_1/ 
 *
 * sarthak7u@gmail.com
 */

import 'dart:io';

void main() {
  print("What is your name?");
  String? name = stdin.readLineSync();
  print("What is your age?");
  String? age = stdin.readLineSync();
  print("What is your user name?");
  String? uname = stdin.readLineSync();

  print(
      "your name is ${name}, you are ${age} years old, and your username is ${uname}");
}
